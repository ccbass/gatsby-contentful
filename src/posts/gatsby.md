---
title: "Basic Gatsby Tutorial"
date: "2021-08-28"
---

Here's the tutorial markdown post!

## Topics

1. Gatsby
2. GraphQL
3. React
