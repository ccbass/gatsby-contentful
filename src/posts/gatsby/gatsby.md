---
title: "Basic Gatsby Tutorial"
date: "2021-08-28"
---

Here's the tutorial markdown post!

![grass image](./grass.jpg)
## Topics

1. Gatsby
2. GraphQL
3. React
