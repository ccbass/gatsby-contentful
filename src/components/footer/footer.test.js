import React from "react"
import renderer from "react-test-renderer"
import { useStaticQuery } from "gatsby"
import Footer from "./footer"


beforeEach(() => {
  jest.clearAllMocks();

  useStaticQuery.mockImplementationOnce(() => ({
      site: {
        siteMetadata: {
          author: `CCB`,
        },
      },
  }))
})

describe("Footer Tests", () => {
  it("Renders as expected vs snapshot.", () => {
    const footer = renderer.create(<Footer />).toJSON()
    expect(footer).toMatchSnapshot()

  })

})
