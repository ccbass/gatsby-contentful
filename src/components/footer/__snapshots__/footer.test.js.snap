// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Tests Renders as expected vs snapshot. 1`] = `
<footer>
  <p>
    Created by 
    CCB
    , Copyright 2021
  </p>
</footer>
`;
